### PR TITLE
Add more tracing sections to startup

### PR DESCRIPTION
**Pull request by bidetofevil**
## Goal

Add more tracing sections to startup workflow to better profile SDK startup. 

A misleading log line was also removed regarding loading native files

### CODE REVIEW

**Comment by bidetofevil** [comment]
Current dependencies on/for this PR:
* **#467** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/467?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/467?utm_source=stack-comment).

File: embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/DataCaptureServiceModule.kt [hunk 1]
```diff
@@ -95,19 +96,23 @@ internal class DataCaptureServiceModuleImpl @JvmOverloads constructor(
...
-        if (configService.autoDataCaptureBehavior.isPowerSaveModeServiceEnabled()) {
-            EmbracePowerSaveModeService(
-                coreModule.context,
-                backgroundWorker,
-                initModule.clock,
-                systemServiceModule.powerManager
-            )
-        } else {
-            NoOpPowerSaveModeService()
+        Systrace.traceSynchronous("power-service-init") {
```
Comment (fractalwrench) [inline embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/DataCaptureServiceModule.kt:105]: Are these just being added as they're the biggest culprits? I was curious why these services are traced but others aren't

**Review by fractalwrench — APPROVED** [review]
LGTM

File: embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/DataCaptureServiceModule.kt (hunk 1 above)
Comment (bidetofevil) [reply embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/DataCaptureServiceModule.kt:105]: Basically. I added a bunch of them and most aren't bad. I have another PR that needs more testing that starts to improve on some of these, but I'm not comfortable with getting that in so close to a release

**Pull request merged**